index.bs

```diff
@@ -6762,6 +6764,10 @@ Developers desiring more complex and arbitrary matrixing can use a
 {{ConvolverNode}}s and a
 {{ChannelMergerNode}}.
 
+If this node is not [=actively processing=], the output is a single
+channel of silence, even if the diagrams below show two channels of
+output.
```

Comment:
Nit: The diagram is supposed to be "supportive" of the text content. Can we not mention the diagram at all? We can add one more diagram for the case of "single channel of silence", but that's a really low priority.

Reply:
Just added a note that the diagrams show the active processing result.